flake.nix

```
@@ -0,0 +1,12 @@
+{
+  description = "nix-powered automatic disk partitioning";
+
+  inputs.flake-compat = {
+    url = "github:edolstra/flake-compat";
+    flake = false;
+  };
+
+  outputs = { self, nixpkgs, ... }: {
+    lib = import ./lib {inherit (nixpkgs) lib;};
```

```suggestion
    lib = import ./. {inherit (nixpkgs) lib;};
```
Only export `config`, `create` and `mount`